packages/inngest/src/components/connect/index.ts

```diff
@@ -129,12 +138,27 @@ class WebSocketWorkerConnection implements WorkerConnection {
     });
   }
 
-  private get functions(): InngestFunction.Any[] {
-    return (
-      (this.options.functions as InngestFunction.Any[]) ??
-      this.inngest["localFns"] ??
-      []
-    );
+  private get functions(): Record<
+    string,
+    {
+      client: Inngest.Like;
+      functions: InngestFunction.Any[];
```

Comment:
Could this also be `InngestFunction.Like[]`? Or do you need some of the private accesses?

Reply:
getConfig() 😭